core/src/com/unciv/models/ruleset/unique/Countables.kt

```diff
@@ -181,38 +189,25 @@ enum class Countables(
     open val documentationHeader get() =
         "`$text`" + (if (shortDocumentation.isEmpty()) "" else " - $shortDocumentation")
 
-    override fun getErrorSeverity(parameterText: String, ruleset: Ruleset): UniqueType.UniqueParameterErrorSeverity? {
+    /** Leave this only for Countables without any parameters - they can rely on [matches] having validated enough */
+    override fun getErrorSeverity(parameterText: String, ruleset: Ruleset): UniqueType.UniqueParameterErrorSeverity? = null
+
+    override fun getDeprecationAnnotation(): Deprecated? = declaringJavaClass.getField(name).getAnnotation(Deprecated::class.java)
+
+    protected fun UniqueParameterType.getTranslatedErrorSeverity(parameterText: String, ruleset: Ruleset): UniqueType.UniqueParameterErrorSeverity? {
+        val severity = getErrorSeverity(parameterText.getPlaceholderParameters().first(), ruleset)
```

Comment:
Not super fond of this since it means we can't have Countables with 2 parameters, but it'll do for now